homeassistant/auth/__init__.py

```diff
@@ -448,13 +453,19 @@ def async_create_access_token(
                 "iss": refresh_token.id,
```

Comment:
Missing a check before this call that verifies that the remote ip being used, is allowed to be used for this credential.